ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    REGION = "us-central1"  # @param {type:"string"}
</pre><p>Change to:</p><p> </p><p>REGION = "[your-region]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>